core/tests/test_utils.py

```diff
@@ -194,14 +236,24 @@ def _restore_stashed_user_env(self):
         for key in self.stashed_user_env:
             os.environ[key] = self.stashed_user_env[key]
 
-        self.stashed_user_env = None  # pylint: disable=attribute-defined-outside-init
+        self.stashed_user_env = None  
+        # pylint: disable=attribute-defined-outside-init
```

Comment:
Actually, this should go on the previous line, otherwise the disabling will apply for all lines below this one. For Python pragmas like this, it is OK to exceed 80 chars in the line.

Alternatively, a better way to handle this might be to just add an `__init__()` to the class and declare this there. (That way we won't even need the pragmas here and in _stash_current_user_env() above.) Don't forget to call the `__init__()` of the superclass as well when doing that.